cfme/tests/containers/test_containers_default_project_replicators.py

```diff
@@ -22,23 +22,32 @@
 REPLICATORS_RELATIONSHIPS_FIELDS = ['Containers Provider', 'Project', 'Pods', 'Nodes',
                                     'My Company Tags']
 
+
```

Comment:
in general i dont understand what are we testing here - are we just making sure titles are ok? what about values?